ftw/simplelayout/aliasblock/hooks.py

```diff
@@ -0,0 +1,34 @@
+from Products.CMFCore.utils import getToolByName
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+import logging
+import pkg_resources
+import re
+
+
+IS_PLONE_5 = pkg_resources.get_distribution('Products.CMFPlone').version >= '5'
+PROFILE_ID = 'profile-ftw.book:default'
```

Comment:
book? and PROFILE_ID is never used anyway

Reply:
Good catch!

Reply:
Fixed that and the unused imports in bd67805